nixos/tests/gitlab.nix

```diff
@@ -63,6 +63,8 @@ import ./make-test-python.nix ({ pkgs, lib, ...} : with lib; {
           jwsFile = pkgs.runCommand "oidcKeyBase" {} "${pkgs.openssl}/bin/openssl genrsa 2048 > $out";
         };
       };
+
+      services.postgresql.package = pkgs.postgresql_13;
```

Comment:
Why is `postgresql` not pointing to the latest version?

Reply:
Oh! It is. We will need something like this for the backport, since the default version for stateVersion==21.05 is PostgreSQL 11.

Reply:
Removed that commit